### ISSUE TITLE
No module named 'crowdin_api.api_resources.reports.requests'

### ISSUE THREAD

**Issue by AdrianRibao** [crowdin/crowdin-api-client-python#111]
When I setup the project, with this code:

```
class CrowdinClient(CrowdinClient):
    TOKEN = settings.CROWDIN_TOKEN

client = CrowdinClient()
```

I get the error:

```
ile "venv/lib/python3.10/site-packages/crowdin_api/api_resources/reports/resource.py", line 15, in <module>
    from crowdin_api.api_resources.reports.requests.cost_estimation_post_editing import (
ModuleNotFoundError: No module named 'crowdin_api.api_resources.reports.requests'
```

**Comment by andrii-bodnar** [comment]
Hi @AdrianRibao, could you please specify the version of Crowdin Client you're using?

**Comment by innomaxx** [comment]
Hi @AdrianRibao, please try fix from [this branch](https://github.com/crowdin/crowdin-api-client-python/tree/fix/reports) and inform us it works or not

**Comment by andrii-bodnar** [comment]
Available in [1.12.1](https://github.com/crowdin/crowdin-api-client-python/releases/tag/1.12.1)
